tools/lint-md.js

```diff
@@ -46701,7 +46701,7 @@ var plugins$2 = [
       { yes: "GitHub" },
       { no: "hostname", yes: "host name" },
       { yes: "JavaScript" },
-      { no: "Node", yes: "Node.js" },
+      { no: "Node", yes: "Node.js", ignoreNextTo: "-API" },
```

Comment:
This needs to change upstream in https://github.com/nodejs/remark-preset-lint-node/blob/0b2a13018de6ad9bcd234e4420bba03ae9f42c29/index.js#L66, a new release of that cut and then pulled into here with this file being regenerated via `tools/node-lint-md-cli-rollup`.

Reply:
https://github.com/nodejs/remark-preset-lint-node/pull/173

Reply:
Upstream change landed, new release of package cut, and PR landed in core to update the lint script. I've rebased and force-pushed so this change should now be outdated, and I think this PR is ready to go.